components/cli/cmd/cellery/logs.go

```diff
@@ -39,21 +41,21 @@ func newLogsCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			isCellValid, err := regexp.MatchString(fmt.Sprintf("^%s$", constants.CELLERY_ID_PATTERN), args[0])
+			isCellValid, err := regexp.MatchString(fmt.Sprintf("^%s$", constants.CelleryIdPattern), args[0])
 			if err != nil || !isCellValid {
 				return fmt.Errorf("expects a valid cell name, received %s", args[0])
 			}
 			if component != "" {
-				isComponentValid, err := regexp.MatchString(fmt.Sprintf("^%s$", constants.CELLERY_ID_PATTERN), component)
+				isComponentValid, err := regexp.MatchString(fmt.Sprintf("^%s$", constants.CelleryIdPattern), component)
 				if err != nil || !isComponentValid {
 					return fmt.Errorf("expects a valid component name, received %s", args[0])
 				}
 			}
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := commands.RunLogs(args[0], component, syslog); err != nil {
-				return fmt.Errorf("error running cellery log, %v", err)
+			if err := instance.RunLogs(cli, args[0], component, syslog); err != nil {
+				util.ExitWithErrorMessage("Cellery logs command failed", err)
```

Comment:
Why are we exiting with error if we are using RunE here?

Reply:
Updated. Not using RunE, instead using util.ExitWithErrorMessage.